contributors/design-proposals/local-storage-overview.md

```diff
@@ -512,7 +523,7 @@ Note: Block access will be considered as a separate feature because it can work
         requests:
           storage: 80Gi
     ```
-4. It is also possible for a PVC that requests `volumeType: file` to also use a block-based PV.  In this situation, the block device would get formatted with the filesystem type specified in the PV spec.  And when the PV gets destroyed, then the filesystem also gets destroyed to return back to the original block state.
+4. The block device would get formatted with the filesystem type specified in the PV spec.  And when the PV gets destroyed, then the filesystem also gets destroyed to return back to the original block state.
```

Comment:
Yes, user should be able to handle this.  It's not clear yet whether kubelet should format it or not.  My understanding is if fsType is given in PVC then user would expect it be formatted by kubernetes; otherwise they'll just get a raw block device.

The whole block storage is under active discussion, I'm only updating the obvious part here.